packages/admin/cms-admin/src/dam/DataGrid/fileUpload/useDamFileUpload.tsx

```diff
@@ -55,6 +56,7 @@ type ImportSource = { importSourceType: never; importSourceId: never } | { impor
 
 interface UploadFilesOptions {
     folderId?: string;
+    // deprecated: add it directly to file props instead
```

Comment:
Please change this to a JSDoc comment using `@deprecated`

Reply:
19b4f456fea978097c56f6f85936b41677de46c2